lib/mruby/build/command.rb

```diff
@@ -162,6 +188,46 @@ def get_dependencies(file)
       end
       deps << MRUBY_CONFIG
     end
+
+    def test_code_template(function: nil, header: nil)
```

Comment:
This function is kind of ugly. I didn't know how far back we support Ruby so I didn't use the tilde `<<~` form of heredocs to make it a bit less ugly.